Libraries/Microsoft.Teams.Extensions/Microsoft.Teams.Extensions.Graph/Microsoft.Teams.Extensions.Graph.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved.-->
+<!-- Licensed under the MIT License.-->
+
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <Import Project="..\..\Directory.Build.props" Condition="'$(Version)' == ''" />
+
+    <PropertyGroup>
+        <PackageId>Microsoft.Teams.Extensions.Graph</PackageId>
+        <PackageDescription>Teams Context Extensions</PackageDescription>
+        <PackageProjectUrl>https://microsoft.github.io/teams-ai</PackageProjectUrl>
+        <PackageTags>microsoft;teams;msteams;copilot;ai;extensions;graph;context</PackageTags>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <TargetFrameworks>net8.0;net9.0;netstandard2.0;netstandard2.1</TargetFrameworks>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <LangVersion>latest</LangVersion>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Graph" Version="[5.0.0,)" />
```

Comment:
are you sure this works? I get an error when I try saying "is not a valid version string."

Reply:
but when I use `5.*` it works

Reply:
Yep it work for me I got it from [this](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#version-ranges)